cvmfs/network/download.h

```diff
@@ -288,7 +288,10 @@ struct JobInfo {
   char *info_header;
   z_stream zstream;
   shash::ContextPtr hash_context;
-  int wait_at[2];  /**< Pipe used for the return value */
+
+  /**< Pipe used for the return value */
```

Comment:
I think the correct Doxygen syntax here is
```suggestion
  /// Pipe used for the return value
```